TestAssets/TestPackages/dotnet-dependency-tool-invoker/dotnet-dependency-tool-invoker.csproj

```diff
@@ -1,21 +1,16 @@
-﻿<Project>
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), testAsset.props))\testAsset.props" />
-  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
-
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <VersionPrefix>1.0.0-rc</VersionPrefix>
-    <TargetFramework>$(CliTargetFramework)</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
We should have a property for this, otherwise, when we move to netcoreapp3.1, we will have to do a bunch of manual changes.

This is true for most places.

Reply:
True, but we would have to do it a bit differently, as the TestAssets projects don't import the repo's Directory.Build.props.  So I'd prefer to leave that for later.